node/public/JS/bmi-calc.js

```diff
@@ -0,0 +1,55 @@
+function findstatus(bmio) {
+    if (bmio < 18.5) {
+        return 'underweight';
```

Comment:
Consider using enum/constants for hardcoded strings used at multiple places. typos are common and difficult to catch in untested code